src/core/annotation.js

```diff
@@ -2780,6 +2781,25 @@ class TextWidgetAnnotation extends WidgetAnnotation {
       !this.hasFieldFlag(AnnotationFieldFlag.FILESELECT) &&
       this.data.maxLen !== 0;
     this.data.doNotScroll = this.hasFieldFlag(AnnotationFieldFlag.DONOTSCROLL);
+
+    // Check if we have a date or time.
+    const {
```

Comment:
Would this perhaps fit on a single line, so it's a bit more compact, if formatted like `const { data: { actions } } = this;` ?

Reply:
`gulp lint` formatted it like this.